app/javascript/components/recordings/Recordings.jsx

```diff
@@ -1,64 +1,37 @@
-import DeleteRecordingForm from '../forms/DeleteRecordingForm';
+import SearchBarQuery from '../shared/SearchBarQuery';
+import RecordingsList from './RecordingsList';
 
 export default function Recordings() {
-  const { isLoading, data: recordings } = useRecordings();
+  const [input, setInput] = useState('');
+  const { data: recordings } = useRecordings(input);
   const { refetch: handleRecordingReSync } = useRecordingsReSync();
 
-  if (isLoading) return <Spinner />;
-
   return (
     <div className="pt-3 wide-background full-height-rooms">
-      <Row>
-        <Button className="my-2" onClick={handleRecordingReSync}>Re-sync Recordings</Button>
-        <Card className="border-0 shadow-sm p-0">
-          <Table hover className="text-secondary mb-0">
-            <thead>
-              <tr>
-                <th>Name</th>
-                <th>Length</th>
-                <th>Users</th>
-                <th>Visibility</th>
-                <th>Formats</th>
-              </tr>
-            </thead>
-            <tbody>
-              {recordings.map((recording) => (
-                <tr key={recording.id} className="recordings align-middle">
-                  <td className="text-dark">
-                    <div> <FontAwesomeIcon className="mx-2 mt-4" icon={faVideo} size="2xl" /> <strong> {recording.name} </strong> </div>
-                    <div className="small text-muted ms-5 ps-2"> {recording.created_at} </div>
-                  </td>
-                  <td> {recording.length}min </td>
-                  <td> {recording.users} </td>
-                  <td> {recording.visibility} </td>
-                  <td>
-                    {recording.formats.map((format) => (
-                      <div key={format.id}> {format.recording_type} </div>
-                    ))}
-                  </td>
-                  <td>
-                    <Modal
-                      modalButton={<FontAwesomeIcon className="" icon={faTrashCan} size="lg" />}
-                      title="Are you sure?"
-                      body={<DeleteRecordingForm recordId={recording.record_id} />}
-                    />
-                  </td>
-                </tr>
-              ))}
-            </tbody>
-          </Table>
-        </Card>
-      </Row>
+      <Stack direction="horizontal" className="w-100 mt-4">
+        <SearchBarQuery setInput={setInput} />
+        <Button className="my-2 ms-auto" onClick={handleRecordingReSync}>Re-sync Recordings</Button>
+      </Stack>
+      <Card className="border-0 shadow-sm p-0 mt-4">
+        <Table hover className="text-secondary mb-0">
+          <thead>
+            <tr className="text-muted small">
+              <th>Name</th>
+              <th>Length</th>
+              <th>Users</th>
+              <th>Visibility</th>
+              <th>Formats</th>
+            </tr>
+          </thead>
+          <RecordingsList recordings={recordings} />
```

Comment:
Separating the `tbody` into a separate component seems a little weird to me

Reply:
Not a fan of using tables, personally I would use rows here.

I'd like to keep this as a component as it is re-used in RoomRecordings.

Reply:
Right.